scripts/clean_disk_cache.sh

```diff
@@ -7,6 +7,8 @@
 # Clean out any files in the Bazel disk cache which haven't been used for over
 # thirty days.
 
+set -eu
```

Comment:
Sure. That's because `/bin/bash` usage could not be the best portability solution. If that shebang works on a particular OS, failure on another cannot be discounted.

In case shell arguments still desired next to `bash`, though:

```diff
-#!/usr/bin/env bash
+#!/usr/bin/env -S bash -eu
# Clean out any files in the Bazel disk cache which haven't been used for over
# thirty days
- set -eu
```

But, I'm not quite sure if that feature is a standard of `env` command.

Reply:
I see it on mac and linux. Is there an OS you're not seeing it on?

Reply:
> I see it on mac and linux. Is there an OS you're not seeing it on?

For example, OpenBSD places bash on `/usr/local/bin`.

Reply:
I mean the `-S` flag

Reply:
> I mean the -S flag

The `-S` flag could reduce the script portability. It's available on Linux and some BSD systems (e.g, [FreeBSD](https://www.freebsd.org/cgi/man.cgi?env)). Unavailable, for example, on [OpenBSD](https://man.openbsd.org/env).

If I may suggest,

```bash
set #...
```

would be the simplest way to handle shell parameters.

Reply:
FWIW, I'd avoid -S just to avoid debugging some bizarre failure in the future. But I don't feel strongly.